docs/testing/continuous-integration.md

```diff
@@ -120,85 +141,3 @@ A consequence of this caching is that test jobs for branches which
 will be significantly slower than normal, because they won't get to
 benefit from the cache.
-
-## Travis CI
-
-### Configuration
-
-The main Travis configuration file is
-[.travis.yml](https://github.com/zulip/zulip/blob/master/.travis.yml).
-The specific test suites we have are listed in the `matrix` section,
-which has a matrix of Python versions and test suites (`$TEST_SUITE`).
-
-Currently there is only the production test suite in this section as we
-have moved the backend and frontend suite to CircleCI. So the value of
-the variable `$TEST_SUITE` would be always `production`.
-
-We've configured it to use a few helper scripts for each job:
-
-* `tools/ci/setup-$TEST_SUITE`: This script sets up the test
-  environment for the production suite. This is a complicated process
-  because of all the packages Travis installs.  See the comments in
-  `tools/ci/setup-production` for details.
-* `tools/ci/$TEST_SUITE`: The script that runs the actual test
-  production test suite.
-
-The main purpose of the distinction between the two is that if the
-`setup-production` job fails, Travis CI will report it as the suite
-having "Errored" (grey in their emails), whereas if the `production` job
-fails, it'll be reported as "Failed" failure (red in their emails).
-Note that Travis CI's web UI seems to make no visual distinction
-between these.
-
-An important detail is that Travis CI will by default hide most phases
-other than the actual test; you can see this easily by looking at the
-line numbers in the Travis CI output.  There are actually a bunch of
-phases (e.g. the project's setup job, downloading caches near the
-beginning, uploading caches at the end, etc.), and if you're debugging
-our configuration, you'll want to look at these closely.
-
-### Useful debugging tips and tools
-
-* Zulip uses the `ts` tool to log the current time on every line of
-  the output in our Travis CI scripts.  You can use this output to
-  determine which steps are actually consuming a lot of time.
-
-* For performance issues,
-  [this statistics tool](https://scribu.github.io/travis-stats/#zulip/zulip/master)
-  can give you test runtime history data that can help with
-  determining when a performance issue was introduced and whether it
-  was fixed.  Note you need to click the "Run" button for it to do
-  anything.
-
-* You can [sign up your personal repo for Travis CI][travis-fork] so
-  that every remote branch you push will be tested, which can be
-  helpful when debugging something complicated.
-
-[travis-fork]: ../git/cloning.html#step-3-configure-continuous-integration-for-your-fork
-
-### Performance optimizations
-
-#### Caching
-
-We cache the following as well apart from what is mentioned in CircleCI
-caching section.
-
-* Built/downloaded emoji sprite sheets and data.
-
-This is probably worth eventually adding to the CircleCI caches, but
-because it only saves ~5s, it hasn't been a priority yet.
```

Comment:
Can you open an issue for doing this in CircleCI?  It seems probably worth tracking; even 5s per job with 5 jobs adds up to a significant amount of runtime, and we have all the tooling to do this in the development environment, so it should be relatively easy.